bulbs/special_coverage/static/special_coverage/js/special-coverage-loader.js

```diff
@@ -1,9 +1,7 @@
 module.exports = (function () { // eslint-disable-line no-unused-vars
-  var _ = require('lodash');
-  var assign = _.assign;
-  var defaults = _.defaults;
-  var isUndefined = _.isUndefined;
-  var trim = _.trim;
+  function isUndefined (suspect) {
+    return suspect === undefined; // eslint-disable-line no-undefined
```

Comment:
I think it's better form to do `typeof(suspect) === 'undefined'`, avoids the eslint error probably as well.